src/sources/logplex.rs

```diff
@@ -23,6 +24,7 @@ use warp::http::{HeaderMap, StatusCode};
 #[derive(Deserialize, Serialize, Debug, Clone)]
 pub struct LogplexConfig {
     address: SocketAddr,
+    query_params: Vec<String>,
```

Comment:
Since this will be pulled out as the public facing user configuration, could we call it `query_parameters`?